tests/main/confinement-classic/task.yaml

```diff
@@ -4,7 +4,10 @@ details: |
     classic confinement can be executed correctly. There are two variants of
     this test (classic and jailmode) and the snap (this particular one) should
     function correctly in both cases.
-systems: [-ubuntu-core-16-*,-debian-*]
+systems:
+    - -ubuntu-core-16
+    # No confinement (AppArmor, Seccomp) available on these systems
```

Comment:
But this is classic confinement :) It should just work, except for jailmode. I think we should not disable any such test but instead make the part that exercises confinement dependant on the hidden command that tells us if confinement is enabled or not.